src/cpp/linearstoppotential.h

```diff
@@ -58,6 +58,7 @@ class LinearStop {
                 const Resources &resources);
 
   double getTime() { return time_; }
+  bool getStop() { return stop_called_; }
```

Comment:
You refer to `getStopCalled()` in the Python bindings. I think that's the better name.
```suggestion
  bool getStopCalled() { return stop_called_; }
```

Reply:
Changed `getStop` to `getStopCalled` in second commit!